specifyweb/frontend/js_src/lib/components/Reports/index.tsx

```diff
@@ -216,7 +217,9 @@ function ReportRow({
               />
             </Button.LikeLink>
           </th>
-          <th>{getField(tables.SpReport, 'createdByAgent').label}</th>
+          <th className="link">
+            {getField(tables.SpReport, 'specifyUser').label}
+          </th>
```

Comment:
Can the `className=link` be removed from this Table Header element? This makes the element appear interactive when hovered. 

```suggestion
          <th>{getField(tables.SpReport, 'specifyUser').label}</th>
```

**With className `link`**: 
![image](https://github.com/specify/specify7/assets/64045831/320fe27f-e6e8-48cd-a2b5-5dbc8abd2e71)

**Without className `link`**: 
![image](https://github.com/specify/specify7/assets/64045831/3773567d-e181-4fb5-9d7a-72448c85bf75)

Or is this behavior intentional?

Reply:
Intentional. Notice the heading is gray instead of white (or gray instead of black, in light mode). Rather than change styles globally I made this choice.

Reply:
This heading text color inconsistency appears in the 'Size' heading for record sets as well.

Reply:
> Intentional. Notice the heading is gray instead of white (or gray instead of black, in light mode). Rather than change styles globally I made this choice.

sorry, I don't follow - can you explain?

if the heading is not clickable, then it makes sense that it should appear a bit more grayed out and not have a pointer mouse. or would in this case visual consistency matter more than UX?

also, if you want to still have `link` in there, could you please replace `className="link"` with `className={className.link}` - it won't make a visual difference, but it would make it much easier to find all usages of the "link" class name (for example when testing the impact of a change on all usages of a class name, this would be useful; similarly, when renaming a class name)

Reply:
In this case, visual consistency is more important in my opinion. So in the spirit of compromise, I've added `pointer-events-none` to this heading as well so it doesn't appear to be clickable either!